### PR TITLE
docs: in-code comments for background grid octaves

### PR DESCRIPTION
**Pull request by strasdat**

### CODE REVIEW

File: components/pango_context/src/gl_drawn_plot_background.cpp [hunk 1]
```diff
@@ -16,37 +16,52 @@ struct GlDrawnPlotBackground : public DrawnPlotBackground {
...
+        range_in_graph / params.viewport.range().cast<double>();
+
+    // minimum distance between tics in pixels
+    const double min_dist_in_pixels = 10.0;
+    // minimum distance between tics in graph units
+    Eigen::Array2d tics_min_dist_in_graph =
+        min_dist_in_pixels * graph_units_per_pixel;
+
+    // let s be the number of subdivisions ('grid lines') per octave.
+    double s = u_num_divisions.getValue();
```
Comment (stevenlovegrove) [inline components/pango_context/src/gl_drawn_plot_background.cpp:51]: const?

**Review by stevenlovegrove — APPROVED** [review]
Great, thanks!

**Comment by stevenlovegrove** [comment]
Thanks! Also looks like clang format check is failing? Are you running the pre-commit hook?

**Comment by strasdat** [comment]
@stevenlovegrove not sure what the problem is. I have not seen this before. Shall we try this to get it out-fixed by a bot: https://pre-commit.ci/ ?

**Comment by stevenlovegrove** [comment]
@strasdat the problem was that your PR was not rebased on pango2 so it was the merge-commit that had bad formatting, not your own HEAD, hence why we couldn't see it locally. I've added better debug printing for pre-commit so you can better inspect the issue in the CI build output.

**Pull request merged**